hmpps-kotlin-spring-boot-autoconfigure/src/main/kotlin/uk/gov/justice/hmpps/kotlin/auth/HmppsAuthenticationHolder.kt

```diff
@@ -31,15 +31,6 @@ class HmppsAuthContextFacade {
   val clientId: String?
     get() = authentication?.clientId
 
-  /**
-   * We are gradually moving away from user tokens and instead using client credentials more often.  This will be NONE
-   * if client credentials are used, even if a NOMIS username is passed in.  This means that this method isn't an
-   * adequate test to see if the user is a NOMIS user. The only test at present is to look the user up in the database
-   * too see if they exist.
-   */
-  val authenticationSource: AuthSource
```

Comment:
decided to remove completely as would prefer people to pass usernames through explicitly rather than using the username in the client credentials